solr/core/src/java/org/apache/solr/handler/admin/api/SchemaZkVersionAPI.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.solr.handler.admin.api;
+
+import org.apache.solr.api.EndPoint;
+import org.apache.solr.handler.SchemaHandler;
+import org.apache.solr.request.SolrQueryRequest;
+import org.apache.solr.response.SolrQueryResponse;
+import org.apache.solr.security.PermissionNameProvider;
+
+import static org.apache.solr.client.solrj.SolrRequest.METHOD.GET;
+
+/**
+ * V2 API for checking the ZK version of an in-use schema.
+ *
+ * This API (GET /v2/collections/collectionName/schema/zkversion) is analogous to the v1
+ * /solr/collectionName/schema/zkversion API.
+ */
+public class SchemaZkVersionAPI {
```

Comment:
This isn't really a request for a change, just more like I wish the term `zk` didn't bleed into our public APIs...    Sigh.   The conversion looks good!